lang/japanese.xml

```diff
@@ -315,468 +328,1105 @@
+			<Text>敵よりも[ICON_VisLimited]外交的視野が広い場合は、モンゴルの全ユニットの[ICON_Strength]戦闘ボーナスが通常の倍になる。[ICON_TradeRoute]交易路を作ると、目的地の都市に[ICON_TradingPost]交易施設が即座に設置される。</Text>
+		</Replace>
+
+		<!-- == NORWAY == -->
+		<!-- = civilization ability = -->
+		<Replace Tag="LOC_TRAIT_CIVILIZATION_EARLY_OCEAN_NAVIGATION_DESCRIPTION" Language="ja_JP">
+			<Text>「造船」の技術を研究するとユニットが外洋航行能力を得る。海洋近接戦闘ユニットが中立地帯で回復するようになる(ただし、ロングシップがこの能力を使うには沿岸にいることが必要)。乗船時と下船時に追加の[ICON_Movement]移動コストがかからなくなる。</Text>
+		</Replace>
+		<!-- = leader ability = -->
+		<Replace Tag="LOC_TRAIT_LEADER_THUNDERBOLT_DESCRIPTION" Language="ja_JP">
+			<Text>すべての海洋近接ユニットに対する[ICON_PRODUCTION]生産力が+50%され、それらは沿岸タイルを襲撃できる。[NEWLINE][NEWLINE]聖地に対する[ICON_PRODUCTION]生産力+50%。聖地が沿岸から隣接ボーナスを得る。[NEWLINE][NEWLINE]聖地に隣接する港は隣接ボーナスとして[ICON_GOLD]ゴールド+2を得る。</Text>
+		</Replace>
+		<Replace Tag="LOC_TRAIT_LEADER_THUNDERBOLT_EXPANSION2_DESCRIPTION" Language="ja_JP">
+			<Text>すべての海洋近接ユニットに対する[ICON_PRODUCTION]生産力が+50%され、それらは沿岸タイルを襲撃できる。鉱山を略奪または沿岸襲撃した際、通常の[ICON_GOLD]ゴールドに加えて[ICON_SCIENCE]科学力を得る。採石場、牧草地、プランテーション、またはキャンプを略奪または沿岸襲撃した際、通常の[ICON_FAITH]信仰力に加えて[ICON_CULTURE]文化力を得る。[NEWLINE][NEWLINE]聖地に対する[ICON_PRODUCTION]生産力+50%。聖地が沿岸から隣接ボーナスを得る。[NEWLINE][NEWLINE]聖地に隣接する港は隣接ボーナスとして[ICON_GOLD]ゴールド+2を得る。</Text>
+		</Replace>
+		<Row Tag="LOC_DISTRICT_HOLY_SITE_NORWAY_COAST_FAITH" Language="ja_JP">
+			<Text>隣接する沿岸/湖タイルにより[ICON_Faith]信仰力+{1_num}。</Text>
+		</Row>
+		<Row Tag="LOC_HOLY_SITE_HARBOR_ADJACENCY_DESCRIPTION" Language="ja_JP">
+			<Text>隣接する聖地タイルにより+{1_num} [ICON_Gold]</Text>
+		</Row>
+		<!-- = unique unit = -->
+		<Replace Tag="LOC_UNIT_NORWEGIAN_BERSERKER_DESCRIPTION" Language="ja_JP">
+			<Text>ノルウェー固有のユニット。敵文明のタイル上、あるいは乗船中にターンを開始した場合[ICON_Movement]移動力+2。攻撃時に[ICON_Strength]戦闘力+10。[ICON_FAITH]信仰力によって購入できる。</Text>
+		</Replace>
+		<Replace Tag="LOC_ABILITY_BERSERKER_RAGE_DESCRIPTION" Language="ja_JP">
+			<Text>攻撃時に [ICON_Strength] 戦闘力+10。</Text>
+		</Replace>
+		<!-- = unique building = -->
+		<Replace Tag="LOC_BUILDING_STAVE_CHURCH_DESCRIPTION" Language="ja_JP">
+			<Text>ノルウェー固有の建造物。[ICON_FAITH]信仰力を使って使徒と審問官を購入するのに必要。この都市の資源タイルに[ICON_FAITH]信仰力+1。この都市内の沿岸資源タイルに[ICON_PRODUCTION]生産力+1。</Text>
+		</Replace>
+
+		<!-- == NUBIA == -->
+		<!-- = civilization ability = -->
+		<Replace Tag="LOC_TRAIT_CIVILIZATION_TA_SETI_DESCRIPTION" Language="ja_JP">
+			<Text>長距離戦闘ユニットへの[ICON_Production]生産力+30%。すべての長距離戦闘ユニットは戦闘経験値+50%。戦略資源の鉱山の生産力[ICON_Production]+1。ボーナス資源と高級資源の鉱山のゴールド [ICON_Gold] +2。</Text>
+		</Replace>
+		<!-- = unique improvement = -->
+		<Replace Tag="LOC_IMPROVEMENT_PYRAMID_DESCRIPTION" Language="ja_JP">
+			<Text>
+        石工術の技術によって解禁される改善。草原、平原、砂漠、丘陵の砂漠あるいは氾濫原にしか建設できない。[ICON_FOOD]食料+1、[ICON_Faith]信仰力+2。都心に隣接している場合[ICON_Food]食料+2。隣接する区域によって追加の産出を得る。[NEWLINE]キャンパスに隣接するごとに、[ICON_SCIENCE]科学力+2。[NEWLINE]工業区域に隣接するごとに、[ICON_PRODUCTION]生産力+2。[NEWLINE]劇場区域に隣接するごとに、[ICON_CULTURE]文化力+2。[NEWLINE]聖地に隣接するごとに、[ICON_FAITH]信仰力+2。[NEWLINE]商業区域や港に隣接するごとに、[ICON_GOLD]ゴールド+2。[NEWLINE]他のヌビアのピラミッドの隣には建設できない。</Text>
+		</Replace>
```

Comment:
Please check tags - <Text>words</Text> should be on same line.
Same as previous: It doesn't break file and not necessary now, but it doesn't according to file style.